src/platforms/dotnet/guides/maui/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Multi-platform App UI (MAUI)
+description: "Learn about Sentry's .NET integration with .NET Multi-platform App UI (MAUI)."
+redirect_from:
+  - /platforms/dotnet/maui/
+---
+
+Sentry has an integration for the [.NET Multi-platform App UI (MAUI)](https://dotnet.microsoft.com/apps/maui) through the [Sentry.Maui NuGet package](https://www.nuget.org/packages/Sentry.Maui).
+
+## Installation
+
+Add the Sentry dependency:
+
+```powershell {tabTitle:Package Manager}
+Install-Package Sentry.Maui -Version {{ packages.version('sentry.dotnet.maui') }}
+```
+
+```shell {tabTitle:.NET Core CLI}
+dotnet add package Sentry.Maui -v {{ packages.version('sentry.dotnet.maui') }}
+```
+
+This package extends [Sentry.Extensions.Logging](/platforms/dotnet/guides/extensions-logging/). This means that besides the MAUI related features, through this package you'll also get access to all the framework's logging integration and also the features available in the main [Sentry](/platforms/dotnet/) SDK.
+
+### Configure
+
+In your `MauiProgram.cs` file, call `UseSentry` on your `MauiAppBuilder`, and include any options you would like to set.  The `Dsn` is the only required parameter.
+
+For example:
+
+```csharp
+public static MauiApp CreateMauiApp() =>
+    MauiApp.CreateBuilder()
+        .UseMauiApp<App>()
+        .UseSentry(options =>
+        {
+            options.Dsn = "___PUBLIC_DSN___";
+        })
+        // The remainder of your configuration
+        .Build();
+}
+```
+
+## Preview Status
+
+The `Sentry.Maui` SDK is currently in public preview.  All APIs added by this package are subject to breaking changes.  We appreciate your feedback, and would love to hear from you!
+
+- [Open Issues (GitHub)](https://github.com/getsentry/sentry-dotnet/issues?q=is%3Aissue+is%3Aopen+label%3AMAUI)
+- [New Issues or Feature Requests (GitHub)](https://github.com/getsentry/sentry-dotnet/issues/new/choose)
+- [Community Chat (Discord)](https://discord.com/channels/621778831602221064/621783515423440927)
+
+This documentation will be improved as we get closer to GA release.  Thanks for your patience!
```

Comment:
Improvement is subjective, so we want to avoid saying that, but we can say objectively that the documentation will be updated

```suggestion
This documentation will be updated as we get closer to general availability release. 
```